cernopendata/modules/fixtures/data/articles/cms-physics-objects-2010/cms-physics-objects-2010.md

```diff
@@ -0,0 +1,153 @@
+    *   particle-ﬂow relative isolation: sum of the p<sub>T</sub> of all charged hadrons and of the transverse energy of all neutral hadrons and photons reconstructed by the particle-flow algorithm in a cone of radius ΔR < 0.4 centred on the muon track direction.
+
+
+<br>
+
+
+## <div id="jetmet2010"> Jets and missing transverse energy (MET) <img src="../../../../static/articles/cms-physics-objects-2010/jet.png" width="10%" align="right"> </div>
+
+(N.B.: b-jets described separately [below](#bjet2010))
+
+The most commonly used variants of jets and missing E<sub>T</sub> are:
+
+*   `recoPFJets_ak5PFJets__RECO` (collection of objects of type `reco::PFJet`)
+*   `recoPFMETs_pfMet__RECO` (collection of objects of type `reco::PFMET`)
+
+The jets typically require Jet Energy Correction (JEC) to be applied, and the preference for MET is to apply so-called type-I corrections, which propagate the JEC to MET. The jets should have a JEC stored, although this is often re-run for analysis with the latest version of JEC constants. The 2010 AOD should have a reasonable JEC, although there may have been slight changes to JEC in later re-reco versions that we have not propagated all the way.
+
+For most educational purposes, the PF jets and MET are ready to use after JEC and type-I, since the corrections are relatively small (typically < 5% and mostly < 10%) and the changes in the corrections would have been even smaller (typically < 2% level).
+
+
+#### Jets — Things to remember
+
+1. JEC: jets should be corrected for JEC, the knowledge of which is the dominant jet systematic; the JEC uncertainties may also be available in AOD
+2. JetID: jet quality cuts should be used to avoid detector problems; there should be tight and loose flags for this
+3. JER: jet results should be unfolded for jet p<sub>T</sub> resolution (JER), which varies between 5–15%; if there is an accompanying MC sample, the MC JER should be oversmeared typically by 10% or more with similar uncertainty; if not, it would be useful to provide parameterised JER
+
+
+#### MET — Things to remember
+
+1. Quality flags: MET should be cleaned for detector problems; there should be several flags for these
+2. Type-I: whether or not JEC is propagated to MET, the JEC is an important systematic
+3. Important uncertainties are JER, JES and unclustered energy: the JER resmearing (uncertainty on JER per jet: 5–20%) and JES systematics should be propagated to MET, and unclustered energy should be considered as an extra systematic by scaling it up and down by 10%.
+4. For SUSY-type analysis, MET tails often come from jets (ECAL holes, heavy flavour, JER), so analyses typically apply cuts on Δφ(jet, MET) to ensure MET is not aligned with one of the jets
+
+<br>
+
+
+## <div id="bjet2011"> b-jets <img src="../../../../static/articles/cms-physics-objects-2011/jet_b.png" width="10%" align="right"> </div>
```

Comment:
I think pictures can work without `../../..` :) you can only say `/static/path`
I changed it in the more recent files

Reply:
OK

Reply:
Fixed.